### DISCUSSION TITLE
Transparent theme

### DISCUSSION THREAD

**Discussion by williamhorning** [giscus/giscus#107] [Ideas]
On my website, the background is an image and each element's background is transparent so you can see the image. At the moment, giscus sticks out because it is using a black background. A transparent theme would really help for websites like this.
![image](https://user-images.githubusercontent.com/68466727/121442697-2c282480-c95a-11eb-9823-7e6996e82579.png)

**Comment by laymonage** [comment]
Hey @wgyt, yeah this would be awesome. You can try modifying one of the existing themes, e.g. [dark.css](https://github.com/laymonage/giscus/blob/main/styles/themes/dark.css). Only a few of the variables need to be modified, though. I haven't found a way to remove unused variables reliably (#48).
